eng/Subsets.props

```diff
@@ -320,7 +321,7 @@
                              $(CoreClrProjectRoot)tools\dotnet-pgo\dotnet-pgo.csproj;
                              $(CoreClrProjectRoot)tools\aot\ILCompiler\repro\repro.csproj;
                              $(CoreClrProjectRoot)tools\r2rtest\R2RTest.csproj" Category="clr" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
-    <ProjectToBuild Include="$(CoreClrProjectRoot)tools\aot\crossgen2\crossgen2.csproj" Category="clr" />
+    <ProjectToBuild Condition="'$(Crossgen2Supported)' == 'true'" Include="$(CoreClrProjectRoot)tools\aot\crossgen2\crossgen2.csproj" Category="clr" />
```

Comment:
https://github.com/dotnet/runtime/pull/84781 disabled running crossgen2 for riscv. It should be done in the same place.

Reply:
After some thinking I'm not so sure that this would be enough to just make changes in src/coreclr/crossgen-corelib.proj. Don't know what Risk-V guys did, but in my case I was able to build whole runtime + almost run all tests using standards method. 

Will try to resurrect work. When I file issue, it was very ideal, since I have LA64 .NET 7 SDK and now it would require some jumping from the hoops on my side 😢